doc/sphinx/source/utils/RTW/user_guide/quick_start.rst

```diff
@@ -0,0 +1,42 @@
+.. _quick_start_guide:
+
+Quick Start Guide
+=================
+
+.. include:: ../common.txt
+
+* Checkout the |RTW|::
```

Comment:
Clone the ESMValTool repository? I was expecting another kind of command when I read "Checkout the RTW"